CrazyCanvas/engine_config.json

```
@@ -0,0 +1,14 @@
+{
+    "WindowSize": [
+        300,
+        150
```

Since this is going in to master a "normal" resolution should be set for everything that isn't your testing projects (client and server)
```suggestion
        1920,
        1080
```